### PR TITLE
FriendlyId slugs for Tags

### PR DESCRIPTION
**Pull request by mattpolito**
Addresses #139

### CODE REVIEW

File: config/initializers/friendly_id.rb [hunk 1]
```diff
@@ -1,88 +1,22 @@
...
-#
-# Use this to set up shared configuration options for your entire application.
-# Any of the configuration options shown here can also be applied to single
-# models by passing arguments to the `friendly_id` class method or defining
-# methods in your model.
-#
-# To learn more, check out the guide:
-#
-# http://norman.github.io/friendly_id/file.Guide.html
+module FriendlyIdOverride
```
Comment (olivierlacan) [inline config/initializers/friendly_id.rb]: I appreciate the idea here but don't remove default configuration comments. Comments like this are extremely helpful when doing gem upgrades over time, notably when you need to diff for deprecated configuration options.

Thanks for extracting those slug generation checks.

Comment (mattpolito) [inline config/initializers/friendly_id.rb]: OK.. I was out of town but will probably have time to make this adjustment today.

**Comment by mattpolito** [comment]
PR updated

**Comment by olivierlacan** [comment]
We'll probably need to add a rake task (or maybe a migration would be best) to automatically update the slugs on old tags with `tags.find_each { |t| t.save! }`, no?

**Comment by mattpolito** [comment]
I was thinking that same thing. Wasn't sure how you wanted to handle it,
but it is an issue.

--
Matt Polito
904.534.0664

**Comment by olivierlacan** [comment]
Another issue on try.orientation.io right now:

image

**Comment by mattpolito** [comment]
That is because the tag has no slug yet. Needs to be re-saved to have one generated. That would be covered with the task.

**Comment by olivierlacan** [comment]
Ok I'll add a migration then because we can't ask people to check for rake tasks at this point.

**Comment by olivierlacan** [comment]
@mattpolito Alright, done. https://github.com/orientation/orientation/commit/389fb0423ebd8db9aa972976977f0cd5ef3de704

Slight issue is the reserved keyword stuff. I think we need to fix that.

**Pull request merged**